torch/fx/experimental/graph_manipulation.py

```diff
@@ -122,19 +123,17 @@ def serialize_weight(tensor: torch.Tensor) -> Dict:
 
 
 def serialize_leaf_module(
-    mod: torch.nn.Module, weights_metadata: Dict, weights: Dict, name_prefix: str
+    node: Node, weights_metadata: Dict, weights: Dict, name_prefix: str
 ) -> Dict:
     parameters: Dict[str, Any] = {}
-    parameters["name"] = type(mod).__name__
-    for name, buffer in mod.named_buffers():
-        weights_metadata[f"{name_prefix}.{name}"] = serialize_weight(buffer)
-        weights[f"{name_prefix}.{name}"] = buffer
-    for name, parameter in mod.named_parameters():
-        weights_metadata[f"{name_prefix}.{name}"] = serialize_weight(parameter)
-        weights[f"{name_prefix}.{name}"] = parameter
-    if isinstance(mod.__constants__, List):
-        for constant in mod.__constants__:
-            parameters[constant] = str(getattr(mod, constant))
+
+    for p_name, p_value in node.attrs_for_lowering.items():  # type: ignore
```

Comment:
Why is there a `# type: ignore` here?

Reply:
Oh this is because I'm doing monkey-patch to add `attrs_for_lowering` which would cause the mypy test fail with error `node doesn't has attribute attrs_for_lowering`.